data/strings/strings.txt

```diff
@@ -25383,3 +25383,26 @@
     it = https://organicmaps.app/it/
     ru = https://organicmaps.app/ru/
     tr = https://organicmaps.app/tr/
+
+  [osm_wiki_about_url]
+    comment = Link to OSM wiki for About Page
```

Comment:
Looks like it's used in 3 different places, does it make sense to mention all of them here?

Reply:
Strings are fixed in regenerated.
But there is a failure in iOS build: 
`/Users/runner/work/organicmaps/organicmaps/iphone/Maps/UI/Editor/MWMEditorNotesFooter.m:25:30: error: string literal must be prefixed by '@'`
Probably there is some typo in your suggestion for iOS change here: https://github.com/organicmaps/organicmaps/pull/4844#issuecomment-1482503908 
?

Reply:
🤦 yes, of course: `L(@"string")`

Reply:
Thanks, I fixed it and code compiles now: https://github.com/cyber-toad/organicmaps/actions/runs/4526319754